README.rst

```diff
@@ -162,8 +162,19 @@ Setup
     go_rules_dependencies()
     go_register_toolchains()
 
-  You can add more external dependencies to this file later (see
-  `go_repository`_).
+* You can add more external dependencies to this file later (see
+  `go_repository`_). This rule is in Gazelle and will require you to load it.
+
+  .. code:: bzl
+
+  load("@bazel_gazelle//:deps.bzl", "go_repository")
```

Comment:
Ideally, this should be a complete, pasteable example. Include the `http_archive` for `bazel_gazelle` here, and make sure to include the load and call of `gazelle_dependencies`.